src/Types/Duration.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Laudis Neo4j package.
+ *
+ * (c) Laudis technologies <http://laudis.tech>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Laudis\Neo4j\Types;
+
+use Bolt\structures\Duration as BoltDuration;
+
+class Duration
+{
+    private int $months;
```

Comment:
1. Great suggestion regarding `toDateInterval`. Tasked it.
2. About duration, i haven't used them either but i've based my (simple) parsing implementation on the [Bolt\structures\Duration](https://github.com/neo4j-php/Bolt/blob/master/src/structures/Duration.php) implementation which only contains the values present here. How do you suggest we proceed here? (off the top of my mind and since i haven't worked on the low-level of things, i am not sure whether we'd be able to tap into the raw result as we did with nodes, and parse that into a fully-fledged object instead?!)

Reply:
Hmm, we might have to create a PR on stefanaks' project. I'll take an in depth look into this tomorrow, thank you for bringing this to our attention :)